README.md

```diff
@@ -132,7 +135,9 @@ This command has two required flags:
 
 `--private-key` - loadbalancers wallet private key (string representation of hex value prefixed with 0x), used for sending founds on payout
 
-`--payout-reward` - defined total reward amount that will be distributed on the payout (amount in Planck)
+`--payout-reward` - defined total reward amount that will be distributed on the payout (amount in Planck). If omitted entire balance of lb wallet will be used as total reward. If this options is selected than `--lb-payout-fee-address` must be set
+
+`--lb-payout-fee-address` - address on which load balancer fee will be sent. If omitted, load balancer fee will be left on load balancer wallet after payout. This flag is **required** if `--payout-reward` is set to 0
```

Comment:
It's better to send all if -1 or not set at all?

Reply:
0 is the default value (not set), but I will change it to -1 it makes more sense.